packages/labs/task/src/task.ts

```diff
@@ -63,57 +63,77 @@ export interface TaskConfig<T extends ReadonlyArray<unknown>, R> {
   onError?: (error: unknown) => unknown;
 }
 
-// TODO(sorvell): Some issues:
-// 1. When task is triggered in `updated`, this generates a ReactiveElement
-// warning that the update was triggered in response to an update.
-// 2. And as a result of triggering in `updated`, if the user waits for the
-// `updateComplete` promise they will not see a `pending` state since this
-// will be triggered in another update; they would need to
-// `while (!(await el.updateComplete));`.
-// 3. If this is instead or additionally triggered in `willUpdate`, the
-// warning goes away in the common case that the update itself does not change
-// the deps; however, the `requestUpdate` to render pending state  will not
+// TODO(sorvell / justinfagnani): Some issues:
+// 1. With the task triggered in `update`, there is no ReactiveElement
```

Comment:
I'm leaving this comment partially here so that we investigate it more. These may be non-issues.